chef/automate
# Nodemanager updating nodes to missing

### :nut_and_bolt: Description: What code changed, and why?

Updating nodemanager nodes when a chef infra node goes missing. 

### :chains: Related Resources

https://github.com/chef/automate/issues/616

### :+1: Definition of Done
When a node on the client runs page get marked missing the same node should be marked missing in the nodemanager

### :athletic_shoe: How to Build and Test the Change
1. `build components/ingest-service && build components/nodemanager-service && start_all_services`
1. Add an old CCR with` send_chef_run_example`
1. Update the missing node threshold in the node lifecycle https://a2-dev.test/settings/node-lifecycle to 6 hours. This kick off the process of updating the missing nodes.
1. Go to the client runs page to ensure the node is missing. 
1. Run the below command to see the node in the nodemanager. 
```
curl -f --insecure -H "api-token: $(get_admin_token)" https://localhost/api/v0/nodes/search -d '{
    "filters": [
      {"key": "manager_id", "values": [""]},
      {"key": "name", "values": ["insights.B"]}
    ]
  }' | jq
```
1. Ensure the "state" is "MISSING"

### :white_check_mark: Checklist
